BootloaderCorePkg/Stage1B/Stage1B.c

```diff
@@ -655,6 +655,10 @@ ContinueFunc (
   AddMeasurePoint (0x2060);
   ASSERT_EFI_ERROR (Status);
 
+  LdrGlobal = (LOADER_GLOBAL_DATA *)GetLoaderGlobalDataPointer ();

```

Comment:
I know it wasn't being checked before, but do we need to check if this pointer is NULL to avoid a KW hit?

Reply:
The global data pointer is set to near IDT base by line#431
`LdrGlobal      = (LOADER_GLOBAL_DATA *)(UINTN)MemPoolCurrTop;`
 and line#472
`SetLoaderGlobalDataPointer (LdrGlobal);`
So, this pointer will not have NULL. Just in case, let me add ASSERT().

Reply:
ASSERT is added in new PR. If KW hits here, it needs to be disposed as not an issue.